client/src/featureform/register.py

```diff
@@ -2921,7 +2921,7 @@ def register_model(self, name: str, tags: List[str] = [], properties: dict = {})
         return model
 
 
-class ResourceClient(Registrar):
+class ResourceClient:
```

Comment:
how would these changes affect the docs? would it be a breaking change?

Reply:
good question. I made sure to check. The fact that you can use the Resource Client to register shit is pretty much a hidden feature/easter egg. Removing it is fine